### ISSUE TITLE
Make `log_lik` methods correct by obtaining latent parameter draws

### ISSUE THREAD

**Issue by athowes** [epinowcast/epidist#267]
In PR #259 we added a slightly incorrect version of the `log_lik_latent_lognormal` function. This allowed progress on model comparison in the Ebola vignette (#48).

It would be good to make this correct however.

For background see:

* #240
* A [thread](https://discourse.mc-stan.org/t/log-likelihood-for-brms-custom-family-that-depends-on-latent-variables/36415) on the Stan forum summarising the issue

In short I see the two most promissing routes as:

1. Try to put the required `pwindow_raw` and `swindow_raw` draws into some part of the `prep` object
2. Use `...` to pass in the `fit` object then extract the draws

Future investigation required to figure out how to do this!

**Comment by seabbs** [comment]
An easy way to do this is to refactor the approach to latent variables so that pwindow and window are defined via non-linear formulas as in the original vignette - that will in turn mean they are available as part of the prep object

**Comment by athowes** [comment]
Fixed effect by `case_id` requires having case in the data (which it isn't right now).
